### ISSUE TITLE
ungoogled-chromium on iOS

### ISSUE THREAD

**Issue by gottabeleGIT** [Eloston/ungoogled-chromium#1103]
As sort of a spin-off of (and reiteration of my comments from) issue #700, I'd like to request @Eloston to please consider making Ungoogled Chromium for iOS/iPadOS based on the following:
#
The Chrome/Chromium for iOS source code can be found here:
https://chromium.googlesource.com/chromium/src.git/+/master/ios/
And build instructions here:
https://chromium.googlesource.com/chromium/src/+/master/docs/ios/build_instructions.md

Yes, I understand that Ungoogled Chromium most likely won't be able to use the Blink engine... however, I like the design of Chrome on iOS and in some cases I can make a reasonable case performance-wise for iOS Chrome vs Safari... if this could become a reality, I think a lot of iOS/iPadOS users would check it out (myself included, even if we have to sideload it). Although I understand Blink is what makes Ungoogled Chromium different from Safari, I think a browser can be based on WebKit and still be different from Safari (e.g. Epiphany on Linux). Even if not, I think Apple is starting to open up to other browsers and the iOS 14 feature of choosing your default browser is a step in the right direction. Another point one could make is that this would be the ultimate alternative for iPadOS users who use iPads as laptop replacements and would like to use Google Chrome but don't want the Google snooping (after all, iOS is generally considered a private, less-Googly platform).

#

In any case, I look forward to seeing the progress on this development and hopefully eventually being able to use Ungoogled Chromium on iOS!

**Comment by wchen342** [comment]
I think you need to find a developer that has enough time and knowledge to maintain such a version. I am the maintainer of Android version of UC and I started because there was no one to make an Android version back then so I decided to do it myself. I was basically on my own (with some help from the author of `Bromite`) and until now I am still the sole maintainer.
What I am trying to say is for an iOS version to happen there has to be someone who can actually do the work to maintain it. @Eloston himself does not have the knowledge to do this.

**Comment by gottabeleGIT** [comment]
Fair enough... I'll leave it open to anyone who'd be interested in making this fork. I would do it myself if I could but I, too, neither have the time nor the skillset to make it happen unfortunately.

**Comment by jstkdng** [comment]
Don't you need MacOS for developing iOS apps? That is a hard block for a lot of people, unless they are willing to use VMs or setup their own hackintosh.

**Comment by gottabeleGIT** [comment]
> Don't you need MacOS for developing iOS apps? That is a hard block for a lot of people, unless they are willing to use VMs or setup their own hackintosh.

Yeah

**Comment by Jean-Gabin** [comment]
Hey, I would be very interested to bring and maintain a blink version of Chromium for iOS. However I don't really know whether its possible to compile a Blink version of chromium for iOS. Could someone give me advice on how to do it (Is it feasible at all ?) Thanks

**Comment by jstkdng** [comment]
AFAIK all browsers for iOS are just reskins of safari, is that because of the app store guidelines? If so, uc could be ported but only through third party stores in jailbroken devices.

**Comment by 3mil3** [comment]
Alternatively I would suggest hosting the app for iOS on 3rd party stores that don't require jailbreaking like AltStore.

Any progress regarding this topic, also interested in forking&maintaining such a project.

**Comment by wchen342** [comment]
A quick FYI: https://www.macworld.com/article/1434394/ios-17-third-party-app-stores-browsers-payment-systems-camera-nfc.html

**Comment by WmRobt** [comment]
> As sort of a spin-off of (and reiteration of my comments from) issue #700, I'd like to request @Eloston to please consider making Ungoogled Chromium for iOS/iPadOS based on the following:
> 
> The Chrome/Chromium for iOS source code can be found here: https://chromium.googlesource.com/chromium/src.git/+/master/ios/ And build instructions here: https://chromium.googlesource.com/chromium/src/+/master/docs/ios/build_instructions.md
> 
> Yes, I understand that Ungoogled Chromium most likely won't be able to use the Blink engine... however, I like the design of Chrome on iOS and in some cases I can make a reasonable case performance-wise for iOS Chrome vs Safari... if this could become a reality, I think a lot of iOS/iPadOS users would check it out (myself included, even if we have to sideload it). Although I understand Blink is what makes Ungoogled Chromium different from Safari, I think a browser can be based on WebKit and still be different from Safari (e.g. Epiphany on Linux). Even if not, I think Apple is starting to open up to other browsers and the iOS 14 feature of choosing your default browser is a step in the right direction. Another point one could make is that this would be the ultimate alternative for iPadOS users who use iPads as laptop replacements and would like to use Google Chrome but don't want the Google snooping (after all, iOS is generally considered a private, less-Googly platform).
> 
> In any case, I look forward to seeing the progress on this development and hopefully eventually being able to use Ungoogled Chromium on iOS!

I’m fed up with sending my info to google and like what e ungoogled chromium’s done. I’m using the Linux version on my custom hardware open suse computer, and now want it on my iOS phones also.

**Comment by PF4Public** [comment]
> I’m using the Linux version on my custom hardware open suse computer

Does it have [Intel Management Engine](https://en.wikipedia.org/wiki/Intel_Management_Engine#Assertions_that_ME_is_a_backdoor) or [AMD Secure Technology](https://en.wikipedia.org/wiki/AMD_Platform_Security_Processor#Reported_vulnerabilities) or [ARM TrustZone](https://en.wikipedia.org/wiki/ARM_architecture_family#Security_extensions:~:text=In%20practice%2C%20since%20the%20specific%20implementation%20details%20of%20proprietary%20TrustZone%20implementations%20have%20not%20been%20publicly%20disclosed%20for%20review%2C%20it%20is%20unclear%20what%20level%20of%20assurance%20is%20provided%20for%20a%20given%20threat%20model%2C%20but%20they%20are%20not%20immune%20from%20attack.)? Just some starting points if you need to go deeper :)

**Comment by WmRobt** [comment]
Greetings and what a suprise that you ask me about the magic inside. Although I consider myself a well-informed user, I can not handle your questions.
